configure.ac

```diff
@@ -77,7 +77,7 @@ if test x$ac_cv_have_decl_S_IRGRP = xyes ; then
 	AC_DEFINE_UNQUOTED([HAVE_DECL_S_IRGRP],1,[Set to 1 if S_IRGRP is defined.])
 else
 	AC_DEFINE_UNQUOTED([HAVE_DECL_S_IRGRP],0)
-	fi
+fi
 
```

Comment:
Err why? This may not have been your indentation style but its is mine and it is consistent across every shell script in this repo.